tox.ini

```diff
@@ -42,12 +42,9 @@ commands = black --skip-string-normalization .
 [testenv:linters]
 skip_install = true
```

Comment:
wazo-agid use the `basepython = python3.10`
Find the best approach and replicate on other repo with pre-commit :pray: 
(I don't have preference, just use same config everywhere)

Reply:
@DrPyser It was an attempt to ensure consistency. Since the CI will always run it with 3.10 whereas we don't know what the main interpreter will be on the individuals computer. Technically if we force pre-commit to use 3.10 then it will have roughly the same result. It will indeed not directly use the existing environment, but it should treat it as the main Python interpreter.